packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -83,10 +86,12 @@ export const ParticipantListWithHeading = (props: {
     <Stack className={participantListStack}>
       <Stack horizontal horizontalAlign="space-between" verticalAlign="center">
         <Stack.Item grow styles={subheadingStyleThemed} aria-label={title} id={subheadingUniqueId}>
-          {paneTitleTrampoline(
-            title ?? '',
-            /* @conditional-compile-remove(total-participant-count) */ totalParticipantCount
-          )}
+          <h2>
```

Comment:
Should we use `heading` aria role instead?
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role